### ISSUE TITLE
Optimist actions are not FSA-compliant

### ISSUE THREAD

**Issue by neverfox** [mathieudutour/redux-optimist-promise#2]
From flux-standard-action:

```
An action MUST NOT include properties other than `type`, `payload`, and `error`, and `meta`.
```

The redux-optimist project is really proposing an extension to the FSA spec (which is [under discussion](https://github.com/acdlite/flux-standard-action/issues/7)). You have the `isFSA` check, but it would actually fail if you put the `optimist` flag on an action. Your tests don't currently seem to cover that case though. So you either shouldn't claim to be FSA-compliant yet, or you should move the optimist flag to `meta` for the time being.

**Comment by Download** [comment]
Lol I came here to suggest the same thing!

Pressed for time now, but I may come back later to work on a PR to change this. Should be a simple change, right?

**Comment by Download** [comment]
@mathieudutour  Closed without comment? Why?

**Comment by mathieudutour** [comment]
I fixed it in 0.2.1, you now specify the optimist in meta:

```
export function addTodo(text) {
  return {
    type: 'ADD_TODO',
    payload: {
      promise: loadTodoServiceAndReturnPromise(text),
      text
    },
    meta: {
      optimist: true
    }
  };
}
```

I'm working on a better doc.

**Comment by Download** [comment]
Wow that's great @mathieudutour  :+1:
